admin/settings-template-form.php

```diff
@@ -60,6 +60,16 @@
                 <span class="description" ><?php echo esc_html__( "This will include the Google Analytics tracking code to your cookie-wall (this is allowed if anonimised, which this plugin will do automatically as well).", 'll-cookie-wall' ) ?></span>
             </td>
         </tr>
+        <tr>
+            <th scope="row">
+                <label for="custom_css"><?php echo esc_html__( "Custom CSS", 'll-cookie-wall' ) ?></label>
+            </th>
+            <td>
+                <?php wp_editor( $custom_css, 'llcw_custom_css', $css_editor_settings ); ?>
+                <br>
+                <span class="description" ><?php echo esc_html__( "You can add CSS rules here to customize the look of your cookie-wall.", 'll-cookie-wall' ) ?></span>
```

Comment:
You can link the extra information in the description to the label, then this gets read out by a screen reader, together with the label. Otherwise screen reader users could miss this info.

Set up:
`<label for="custom_css" aria-describedby="custom_css_description"><?php echo esc_html__( "Custom CSS", 'll-cookie-wall' ) ?></label>`

`<span class="description" id="custom_css_description"><?php echo esc_html__( "You can add CSS rules here to customize the look of your cookie-wall.", 'll-cookie-wall' ) ?></span>
`

Reply:
👍 
Good idea, you know how it work now :-)